Pollination-apps/create-job.md

```diff
@@ -0,0 +1,161 @@
+# Create a Job
+
+A Job on Pollination can be created using the API client, owner name, project name, and
+recipe inputs.
+
+### API Client
+
+A Pollination API client can be created using a Pollination API key. A user of your app
+can find a Pollination API key in the "Developer Settings" section
+under the "Settings" tab of their profile on Pollination. If they don't have a key,
+they can generate one by entering any name for the key and clicking on the "+" button.
+Once they have this key, they can copy it to the clipboard and paste it back to the
+Pollination-app.
+
+![](../.gitbook/assets/pollination-apps/api_key.png)
+
+### Owner Name
+
+The owner name is the username of the user who is scheduling this job on Pollination.
+This username too can be found in the "Profile Settings" section of the "Settings"
+tab on Pollination. The username is typically followed by a "@" on the "Profile Settings".
+
+![](../.gitbook/assets/pollination-apps/owner.png)
+
+### Project Name
+
+The project name is name of the project the scheduled job will go into. A user can create
+a project on Pollination by clicking on the "+" button on the navigation bar at the top.
+If a project already exists, the user can use that project's name.
+
+![](../.gitbook/assets/pollination-apps/project.png)
+
+### Recipe Inputs
+
+Visit any [recipe](https://app.pollination.cloud/recipes) on Pollination to find out
+the inputs for the recipe
+
+> **_Note:_** The recipe needs to be added to the project before it can be used. A recipe can be
+> added to the project by going to the 'Recipes' on the 'Settings' tab of the project.
+
+![](../.gitbook/assets/pollination-apps/recipe.png)
+
+## Example
+
+We will create a job on Pollination to run the
+[daylight-factor](https://app.pollination.cloud/ladybug-tools/recipes/daylight-factor)
+recipe on the gridbased model that ships with Ladybug Tools.
+
+Install the following libraries first
+
+```python
+pip install streamlit pollination-streamlit
+```
+
+Importing necessary libraries
+
+```python
+import streamlit as st
+from pathlib import Path
+from pollination_streamlit.api.client import ApiClient
+from pollination_streamlit.interactors import NewJob, Recipe
+```
+
+Helper function to create a job
+
+> **_Note:_** The last four arguments are recipe specific inputs.
```

Comment:
I will break this down into two functions and make the `create_job` function reusable for other recipes.

One argument for the recipe and another argument and artifacts as dictionaries.